packages/@react-aria/gridlist/src/useGridListItem.ts

```diff
@@ -64,12 +77,34 @@ export function useGridListItem<T>(props: AriaGridListItemOptions, state: ListSt
     // (e.g. clicking on a row button)
     if (
       (keyWhenFocused.current != null && node.key !== keyWhenFocused.current) ||
-      !ref.current.contains(document.activeElement)
+      !ref.current?.contains(document.activeElement)
     ) {
       focusSafely(ref.current);
     }
   };
 
+  let treeGridRowProps: HTMLAttributes<HTMLElement> = {};
+  let hasChildRows;
+  let hasLink = state.selectionManager.isLink(node.key);
+  if (node != null && 'expandedKeys' in state) {
+    // TODO: ideally node.hasChildNodes would be a way to tell if a row has child nodes, but the row's contents make it so that value is always
+    // true...
+    hasChildRows = [...state.collection.getChildren(node.key)].length > 1;
```

Comment:
wonder if the node information should have a property that conveys if the node has a child row of sorts

Reply:
So at the moment the children of a tree row node is the `<TreeItemContent>` node (aka the wrapper around any interactive elements/title in the row) and the child rows hence the > 1 check.